pkg/feature/raw_resources.go

```diff
@@ -24,13 +24,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/cluster"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/metadata/annotations"
 )
 
 const (
 	YamlSeparator = "(?m)^---[ \t]*$"
 )
 
-func createResources(cli client.Client, objects []*unstructured.Unstructured, metaOptions ...cluster.MetaOptions) error {
+func CreateResources(cli client.Client, objects []*unstructured.Unstructured, metaOptions ...cluster.MetaOptions) error {
```

Comment:
This is not the right name anymore :) I wonder if we should rename it to `ApplyResource` instead to stick to what `kubectl apply` does.

Reply:
renamed 
f247686